app/utils/SplashLoadingScreen.ts

```diff
@@ -1,24 +1,30 @@
-// @ts-nocheck File not migrated fully to TS
-/**
- * Created by jakubniezgoda on 06/13/2017.
- */
-
-const SPLASH_DIV = $('div.splashPage');
+const SPLASH_DIV: HTMLElement | null = document.querySelector('div.splashPage');
```

Comment:
Minor:

The type annotation seems redundant to me, as it's the same as the default one for `querySelector`

```suggestion
const SPLASH_DIV = document.querySelector('div.splashPage');
```